docs/data-sources/organization.md

```diff
@@ -27,6 +27,6 @@ In addition to all arguments above, the following attributes are exported:
 * `multi_factor_auth_required` - (Optional) Flag that indicates whether to require users to set up Multi-Factor Authentication (MFA) before accessing the specified organization. To learn more, see: https://www.mongodb.com/docs/atlas/security-multi-factor-authentication/.
 * `restrict_employee_access` - (Optional) Flag that indicates whether to block MongoDB Support from accessing Atlas infrastructure for any deployment in the specified organization without explicit permission. Once this setting is turned on, you can grant MongoDB Support a 24-hour bypass access to the Atlas deployment to resolve support issues. To learn more, see: https://www.mongodb.com/docs/atlas/security-restrict-support-access/.
 * `gen_ai_features_enabled` - (Optional) Flag that indicates whether this organization has access to generative AI features. This setting only applies to Atlas Commercial and defaults to `true`. With this setting on, Project Owners may be able to enable or disable individual AI features at the project level. To learn more, see https://www.mongodb.com/docs/generative-ai-faq/.
-
+* `skip_default_alerts_settings` - (Optional) Disables automatic alert creation and defaults to `true`. When set to true, no organization level alerts will be created automatically.
```

Comment:
```suggestion
* `skip_default_alerts_settings` - (Optional) Disables automatic alert creation. When set to true, Atlas doesn't automatically create organization-level alerts. Defaults to `true`.
```

Favor active voice and present tense. Default values generally come after the 'core' description.

Reply:
@maastha @lmkerbey-mdb Terraform (and other IaCs) are intentionally defaulting to true to avoid drift changes in the customer state configurations (since alerts would be created as a side-effect). Should we add a small note about to explain why we are defaulting to true?

Reply:
yes I think the 'why' would make sense since we diverge from API behavior which clearly states the default value to be false.